File: app/src/main/java/jensklingenberg/de/tuxoid/model/Element/Destination/Destination.java
package jensklingenberg.de.tuxoid.model.Element.Destination;

import jensklingenberg.de.tuxoid.interfaces.IReachable;
import jensklingenberg.de.tuxoid.model.Element.Element;
import jensklingenberg.de.tuxoid.model.Element.ElementGroup;

/**
 * Created by jens on 18.04.17.
 */

public class Destination extends Element implements IReachable {


    Destination(int type, int z, int y, int x) {
        super(type, z, y, x);
    }


    public ElementGroup getElementGroup() {
        return ElementGroup.Destination;
    }
}

File: app/src/main/java/de/jensklingenberg/tuxoid/model/element/destination/Destination.kt
package de.jensklingenberg.tuxoid.model.element.destination

import de.jensklingenberg.tuxoid.interfaces.IReachable
import de.jensklingenberg.tuxoid.model.element.Element
import de.jensklingenberg.tuxoid.model.element.ElementGroup

/**
 * Created by jens on 18.04.17.
 */

open class Destination(type: Int) : Element(type) {
    override val elementGroup =  ElementGroup.Destination
    override fun isReachable() = true
}
